src/tms_service.ts

```diff
@@ -110,6 +135,94 @@ export class TMSService extends AbstractEmsService {
     this._config = config;
   }
 
+  /*
+  This method returns an array of objects with the layers and the new
+  layout['text-field'] to apply using map.setLayoutProperty
+  */
+  public static transformLanguageProperty(
+    layer: LayerSpecification,
+    lang: keyof typeof TMSService.SupportedLanguages
```

Comment:
Since we're handling unrecognized `lang` in this method, I think we can change the argument type to `string`. It's easier to work with in Kibana that way.

```suggestion
    lang: string
```

Reply:
Done with the change to an array for `SupportedLanguages`.